### PR TITLE
Adding filter settings documentation

### PR DESCRIPTION
**Pull request by XDki113r**
Documents the list of filter settings.
A little formatting in the filters.

### CODE REVIEW

**Comment by Brayzure** [comment]
Good additions, but I am considering removing this repo entirely, since the new site will have the new documentation.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Utilibot is an application that aims to be easy to use, but exceptionally powerful and configurable.
 
 ### Features  
-- Kicks, bans, and mutes
+- Kicks, bans, mutes, and deletions
```
Comment (Brayzure) [inline README.md]: I'd prefer not to list it here, since these are the actions a mod can take that generate an audit log.

File: README.md [hunk 2]
```diff
@@ -51,6 +52,15 @@ kick users, mute users (requires a valid Muted role), or can be disabled entirel
 
 Their settings can be edited by typing `!?edit <filter name> <new setting>` (example: `!?edit MentionSpam mute`, `!?edit MajorProfanity off`).  
 
+##### Filter Settings
+Each filter can be configured with independant settings. The default setting for each filter is to **delete** the message when triggered. The filter settings are:
+
+- **Delete**: Instantly deletes the message. The author does not get an [Audit Log](#audit-log) for this.
```
Comment (Brayzure) [inline README.md]: I'd prefer this not be highlighted.

File: README.md (hunk 1 above)
Comment (XDki113r) [inline README.md]: My misunderstanding.

**Comment by XDki113r** [comment]
Is there a way I could help with the documentation on the new site? I'd like to help you with this.

**Pull request merged**